tests/conftest.py

```diff
@@ -1,8 +1,9 @@
 import os
-import django
 
+import django
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "mds.server.settings")
+os.environ["MDS_AUTH_SECRET_KEY"] = "secret_for_tests"
```

Comment:
you should set it as default in `mds.server.settings`

Reply:
I have no sensible default value for this parameter. I feel it is safer to not have a default value to force the application administrator to set a correct value (I provide a .env.example file that could be automatically loaded by a configuration library)